libs/dh/core/shell/src/lib/dh-primary-navigation.component.html

```diff
@@ -20,7 +20,7 @@
       transloco("messageArchive")
     }}</watt-nav-list-item>
 
-    <watt-nav-list-item link="/market-participant/actors">{{
+    <watt-nav-list-item link="/market-participant">{{
```

Comment:
suggestion: let's use the full route. Otherwise both `Actors` and `Grid Areas` menu items get highlighted in the sidebar when the `Grid Areas` route is active. 


```suggestion
    <watt-nav-list-item link="/market-participant/actors">{{
```

![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/38f77674-3146-4069-ba40-c3a7d4b23e51)

Reply:
We need to figure out how to highlight actors, when we are on one of the other tabs under that menupoint..

Reply:
Good point. Could be part of a follo-up PR.